src/Configuration.js

```diff
@@ -4,13 +4,13 @@ import path from "path";
 import ConfigurationError from "./ConfigurationError";
 import execSync from "./execSync";
 
-export function fromCWD() {
-  const rootPath = execSync("git rev-parse --show-toplevel");
-
-  const lernaPath = path.join(rootPath, "lerna.json");
-  const lernaJson = JSON.parse(fs.readFileSync(lernaPath));
+export function fromGitRoot(cwd) {
+  const rootPath = execSync("git rev-parse --show-toplevel", { cwd });
+  return fromPath(rootPath);
+}
 
-  const config = lernaJson.changelog;
+export function fromPath(rootPath) {
+  const config = fromPackageConfig(rootPath) || fromLernaConfig(rootPath);
```

Comment:
I might let `lerna.json` have precedence over `package.json` if both are present.  It's more narrowly scoped.